migrations/20200824182434_migrate_pil_licence_no_to_profile.js

```diff
@@ -0,0 +1,35 @@
+exports.up = function(knex) {
+  return Promise.resolve()
+    // get all profilesIds with a pil
+    .then(() => knex('pils').distinct('profile_id'))
+    .then(pilProfiles => {
+      return pilProfiles.reduce((promise, pilProfile) => {
+        return promise
+          .then(() => knex('profiles')
+            .select('id')
+            .where({ id: pilProfile.profile_id })
+            .whereNull('pil_licence_number')
+            .first()
```

Comment:
What is this query for? We use the profile id as the input, and then the only bit of the result we use is the profile id. It seems a little redundant.

Reply:
If the answer is "it checks that the profile doesn't _already_ have a PIL id" then you'll want to set all of them to null in the `rollback`.

Reply:
you are right on both counts, I was actually going to do that but got distracted by something shiny